src/platform/backends/qemu/qemu_vm_process_spec.cpp

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2019 Canonical, Ltd.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "qemu_vm_process_spec.h"
+
+#include "backend_utils.h"
+#include <QHash>
+#include <QString>
+
+namespace mp = multipass;
+
+
+mp::QemuVMProcessSpec::QemuVMProcessSpec(const mp::VirtualMachineDescription& desc, const QString& tap_device_name,
+                                     const QString& mac_addr)
+    : desc(desc), tap_device_name(tap_device_name), mac_addr(mac_addr)
+{
+}
+
+QString mp::QemuVMProcessSpec::program() const
+{
+    return "qemu-system-" + mp::backend::cpu_arch();
+}
+
+QStringList mp::QemuVMProcessSpec::arguments() const
+{
+    auto mem_size = QString::fromStdString(desc.mem_size);
+    if (mem_size.endsWith("B"))
+    {
+        mem_size.chop(1);
+    }
```

Comment:
This will conflict with #633, won't it?

Reply:
@gerboland is it safe to just remove 38-41 there?

Reply:
@gerboland nevermind, I realize now this just changed places, but it is still required